v5_MIGRATION_GUIDE.md

```diff
@@ -324,26 +324,29 @@ class MyActivity : FragmentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
 +       // can initialize clients outside of onCreate if desired
 -       initializeClients()
-+       payPalLauncher = PayPalLauncher() { paymentAuthResult ->
-+           payPalClient.tokenize(paymentAuthResult) { result ->
-+               when(result) {
-+                   is PayPalResult.Success -> { /* handle result.nonce */ }
-+                   is PayPalResult.Failure -> { /* handle result.error */ }
-+                   is PayPalResult.Cancel -> { /* handle user canceled */ }
-+               }          
-+           }
-+       }
++       payPalLauncher = PayPalLauncher() 
     }
     
     // ONLY REQUIRED IF YOUR ACTIVITY LAUNCH MODE IS SINGLE_TOP
     override fun onNewIntent(intent: Intent) {
-        setIntent(intent)
-+       payPalLauncher.handleReturnToAppFromBrowser(requireContext(), intent)
++       handleReturnToAppFromBrowser(intent)
     }
     
     // ALL OTHER ACTIVITY LAUNCH MODES 
     override fun onResume() {
-+       payPalLauncher.handleReturnToAppFromBrowser(requireContext(), requireActivity().intent)
++       handleReturnToAppFromBrowser(requireActivity().intent)
+    }
+    
+    fun handleReturnToAppFromBrowser(intent: Intent) {
+       // fetch stored PayPalPendingRequest.Success 
+       fetchPendingRequest()?.let {
```

Comment:
I think these lines also need ` +` prefix, since a merchant will have to add this code

Reply:
Ah yes good catch - fixed